src/lib.rs

```diff
@@ -873,7 +946,24 @@ impl JsonlToCiff {
             // Sum of tf's in this doc => doc_length
             let mut doc_length = 0i64;
             for (term, score) in jdoc.vector {
-                let tf = score.round() as i32;
+                let tf = if self.quantize {
+                    // 8-bit scalar quantization: map [min_score, max_score] to [1, 256]
+                    // We use 1-256 to avoid zero values which get filtered out
```

Comment:
Here, as well as on the line above, you say 256 but the actual max score is 255.